### PR TITLE
support larger offsets for import raw

### PR DESCRIPTION
**Pull request by LWinterberg**
Resolves: #6253 probably. Can't test because I need to reconfigure my local build environment.

### CODE REVIEW

**Comment by LWinterberg** [comment]
oh wow, apparently I can't use `int64_t` because `long` is defined differently on Linux compared to macOS and Windows, and wxString::toLong gets confused on one platform or the other. I guess this means then that you'll be able to put in a larger number on Linux compared to the other platforms then in case you need to have an offset greater than 8 petabytes  ¯\\\_(ツ)_/¯

File: src/import/ImportRaw.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ class ImportRawDialog final : public wxDialogWrapper {
 // Initial value for Import Raw dialog
 int ImportRawDialog::mEncoding = SF_FORMAT_RAW | SF_ENDIAN_CPU | SF_FORMAT_PCM_16;
 unsigned ImportRawDialog::mChannels = 1;
-int ImportRawDialog::mOffset = 0;
+long ImportRawDialog::mOffset = 0;
```
Comment (namazso) [inline src/import/ImportRaw.cpp:98]: `long` on Windows is 32 bits, so this wouldn't fix the issue

**Comment by namazso** [comment]
You probably want to just use `long long`, which is 64 bits on both platforms, and there exists a wxWidgets function for it.

**Comment by LWinterberg** [comment]
thanks!

**Review by namazso — APPROVED** [review]
I tested the CI artifact for Windows and it works as expected

**Review by grliszas14 — APPROVED** [review]

**Comment by namazso** [comment]
image
Seems like the previous values are filled back incorrectly

**Pull request merged**